src/main/java/frc/robot/subsystems/IntakePivotSubsystem.java

```diff
@@ -52,50 +54,58 @@ public IntakePivotSubsystem() {
 
   @Override
   public void periodic() {
-    pivotPIDControl();
+    // pivotPIDControl();
+    double armFeedforwardVoltage =
+        feedforward.calculate(
+            (pid.getSetpoint().position - IntakeConstants.PIVOT_FEEDFORWARD_OFFSET) * 0.254,
+            pid.getSetpoint().velocity);
```

Comment:
```
```suggestion
 double armFeedforwardVoltage =
        feedforward.calculate(
            (pid.getSetpoint().position - Units.rotationsToRadians(IntakeConstants.PIVOT_FEEDFORWARD_OFFSET)) * 0.254,
            pid.getSetpoint().velocity);
```

I think this might be the issue, because the getSetpoint().position should return radians.

Reply:
Try simulating this to see if it's improved at all. I'll try to write some tests as well.